addons/flashlights/CfgVehicles.hpp

```diff
@@ -0,0 +1,45 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            //todo: add flashlight attach actions
+        }; 
+    };
+    
+    class NATO_Box_Base;
+    class EAST_Box_Base;
+    class IND_Box_Base;
+    class FIA_Box_Base_F;
+
```

Comment:
Are we talking about the attach actions? Currently, they would serve no purpose, as I don't have the items set up as lightsources yet (will need help on that). I figured it would be better to leave them out until that's been made.